src/index.js

```diff
@@ -7,43 +7,61 @@ import transformCall from './transformers/call';
 import transformImport from './transformers/import';
 
 
-const defaultBabelExtensions = ['.js', '.jsx', '.es', '.es6'];
-export const defaultExtensions = defaultBabelExtensions;
+const defaultExtensions = ['.js', '.jsx', '.es', '.es6'];
 
 function isRegExp(string) {
   return string.startsWith('^') || string.endsWith('$');
 }
 
-// The working directory of Atom is the project,
-// while Sublime sets the working project to the directory in which the current opened file is
-// So we need to offer a way to customize the cwd for the eslint plugin
-export function manipulatePluginOptions(pluginOpts, cwd = process.cwd()) {
```

Comment:
Same for this one. https://github.com/tleunen/eslint-import-resolver-babel-module/blob/master/src/index.js#L60
If we don't have them, we'll need to export the new function and also fix the eslint plugin